packages/secure_storage/amplify_secure_storage_dart/ffigen.cupertino.coreFoundation.config.yaml

```diff
@@ -0,0 +1,50 @@
+# ffigen config for Apple's CoreFoundation Framework
+# to regenerate, run: dart run ffigen --config ffigen.cupertino.security.config.yaml
+# 
+# This should be run on a Mac with Command Line Tools installed.
+#
+# NOTE: You may need to update the path to the `/Frameworks/` directory depending on
+# your MacOS version.
+
+output: 'lib/src/ffi/cupertino/core_foundation.bindings.g.dart'
+name: 'CoreFoundation'
+description: 'Bindings for the CoreFoundation Framework'
+headers:
+  entry-points:
+    - '/Library/Developer/CommandLineTools/SDKs/MacOSX11.1.sdk/System/Library/Frameworks/CoreFoundation.framework/Headers/CFDictionary.h'
```

Comment:
I'm guessing it doesn't matter which version we use since these libs are not expected to change. I don't have 11.1, though, for ex

```
$ ls /Library/Developer/CommandLineTools/SDKs/
MacOSX.sdk	MacOSX11.sdk	MacOSX12.3.sdk
MacOSX11.3.sdk	MacOSX12.1.sdk	MacOSX12.sdk
```

Maybe just point to `MacOSX.sdk`?

Reply:
Huh. So I don't have `MacOSX.sdk` in that directory anymore. I was having issues with Command Line Tools not being able to update a while ago so I uninstalled it and reinstalled it. 11.1 is all I have now.

I'm thinking `/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/` would be a more reliable location to reference. I have `MacOSX.sdk` & `MacOSX12.3.sdk` (my current OS version). I have updated it to use that location instead.

Reply:
Do we need to follow Xcode beta closely to update the paths? 😅

Reply:
No, I think we found a path that isn't expected to change